Rakefile

```diff
@@ -43,6 +43,8 @@ end
 
 Rake::TestTask.new(integration: :compile) do |t|
   t.libs.push 'lib'
+  t.libs.push 'support'
+  t.libs.push 'integration'
```

Comment:
Added to support the simplified "require" statements in integration_test_case.rb